client/components/homePage/homePage.jsx

```diff
@@ -6,24 +6,15 @@ import MonthlySubscr from './MonthlySubscr.jsx';
 import Container from 'react-bootstrap/Container';
 
```

Comment:
we don't need monthly subscription anymore on the home page? and the about section?

Reply:
Making the about page now